utils/dump_leaderboard.go

```diff
@@ -42,12 +45,25 @@ type StarJSON struct {
 type Member struct {
 	Name  string
 	Stars int
+	Ranks [25][2]string
```

Comment:
Add comment explaining the indices? Also, what's in the value? i.e. say what's in the string. Also I think you mean 26 since you'll be indexing by the day number which is 1-25?

Reply:
Actually I think this is unnecessary given my later comment about moving ranks into the dumper.